proto/blueprint.proto

```diff
@@ -545,6 +545,26 @@ message Card {
    * Indictate where to display the headline.
    */
   optional HeadlinePosition headline_position = 43;
+
+  /**
+   * Define card sizes
+   */
+  enum CardSize {
+    CARD_SIZE_UNSPECIFIED = 0;
+    CARD_SIZE_XX_SMALL = 1;
```

Comment:
@waisingyiu @aoifemcl15 Do we need this size if it's never going to be used by MAPI?